test/test.jl

```diff
@@ -90,7 +90,8 @@ end
 @test isapprox(.1+.1+.1, .3)
 @test !isapprox(.1+.1+.1, .4)
 
+@test_throws UndefVarError test_approx_eq(ones(10),ones(11),1e-8,"a","b")
+@test_throws UndefVarError test_approx_eq(ones(10),zeros(10),1e-8,"a","b")
```

Comment:
Oh no!!! I'm sorry. I'll fix it. Thanks for catching this and reviewing my bad PRs :)